post_installation_configuration/configuring-aws-outposts.adoc

```diff
@@ -0,0 +1,95 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="configuring-aws-outposts"]
+= Extending an AWS VPC cluster into an AWS Outpost
+include::_attributes/common-attributes.adoc[]
+:context: configuring-aws-outposts
+:zone-type: AWS Outposts
+
+toc::[]
+
+After xref:../installing/installing_aws/installing-aws-vpc.adoc#installing-aws-vpc[installing a cluster on AWS into an existing VPC], you can create a compute machine set that deploys compute machines in AWS Outposts.
+AWS Outposts is an AWS edge compute service that enables using many features of a cloud-based AWS deployment with the reduced latency of an on-premise environment.
+For more information, see the link:https://docs.aws.amazon.com/outposts/[AWS Outposts documentation].
+
+//AWS Outposts on {product-title} requirements and limitations
+include::modules/aws-outposts-requirements-limitations.adoc[leveloffset=+1]
+
+[id="aws-outposts-environment-info_{context}"]
+== Obtaining information about your environment
+
+To extend an AWS VPC cluster to your Outpost, you must provide information about your {product-title} cluster and your Outpost environment.
+You use this information to complete network configuration tasks and configure a compute machine set that creates compute machines in your Outpost.
+You can use command-line tools to gather the required details.
+
+//Obtaining information from your {product-title} cluster
+include::modules/aws-outposts-environment-info-oc.adoc[leveloffset=+2]
+
+//Obtaining information from your AWS account
+include::modules/aws-outposts-environment-info-aws.adoc[leveloffset=+2]
+
+[id="aws-outposts-network-config_{context}"]
+== Configuring your network for your Outpost
+
+To extend your VPC cluster into an Outpost, you must complete the following network configuration tasks:
+
+* Create a subnet in your Outpost.
+* Ensure that AWS Classic Load Balancers do not use the Outpost subnet.
+* Ensure that the default Ingress Controller does not use the Outpost subnet.
+* Change the Cluster Network MTU.
```

Comment:
@jeana-redhat i've confirmed that the subnet tag will make CCM to ignore the subnet and not affect ALBO (see comments in this review). I believe we get remove the LB sections from here, and aligned where to add it post-GA.

Reply:
re: 1 - thanks! I had wondered if my order here was logical or not :)

re: 2 - per Slack DM, will address all other feedback before removing any sections so that those are discrete changes :+1: